libsolidity/analysis/NameAndTypeResolver.cpp

```diff
@@ -297,7 +297,7 @@ bool NameAndTypeResolver::resolveNamesAndTypesInternal(ASTNode& _node, bool _res
 			if (!resolveNamesAndTypes(*node, false))
 			{
 				success = false;
-				break;
```

Comment:
I'm not sure why this was here. I will run the whole test suite once the expectations are all updated, if it does not crash, I would propose to keep it like this.